ComponentsIT.testEmbed failing

This test has started failing for us in the last ~48 hours:

~~~
[2024-03-26T21:53:23.281Z] [INFO] [ERROR] Tests run: 6, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 1.332 s <<< FAILURE! - in com.adobe.cq.wcm.core.components.it.http.ComponentsIT
[2024-03-26T21:53:23.282Z] [INFO] [ERROR] testEmbed(com.adobe.cq.wcm.core.components.it.http.ComponentsIT)  Time elapsed: 0.663 s  <<< FAILURE!
[2024-03-26T21:53:23.282Z] [INFO] org.junit.ComparisonFailure: .embed.embed-v1 does not match embed-v1-url-youtube.html expected:<...er="0" height="113" []src="https://www.you...> but was:<...er="0" height="113" [referrerpolicy="strict-origin-when-cross-origin" ]src="https://www.you...>
~~~

So it seems the server is now embedding a reffererpolicy attribute that the test doesn't want to see (as per: https://github.com/adobe/aem-core-wcm-components/blob/main/testing/it/http/src/test/resources/components/embed-v1-url-youtube.html)

--- a/testing/it/http/src/test/resources/components/embed-v1-url-trailing-whitespace.html
+++ b/testing/it/http/src/test/resources/components/embed-v1-url-trailing-whitespace.html
@@ -1,7 +1,7 @@
 <div class="embed-v1 embed">
     <div data-cmp-data-layer="{&#34;embed_v1_url_trailing_whitespace&#34;:{&#34;@type&#34;:&#34;core-component/components/embed-v1&#34;}}"
          id="embed_v1_url_trailing_whitespace" class="cmp-embed">
-        <iframe width="200" height="113" src="https://www.youtube.com/embed/vpdcMZnYCko?feature=oembed" frameborder="0"
+        <iframe width="200" height="113" referrerpolicy="strict-origin-when-cross-origin" src="https://www.youtube.com/embed/vpdcMZnYCko?feature=oembed" frameborder="0"
                 allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen
                 title="Ocean Surface: 10 Hours of Relaxing Oceanscapes | BBC Earth"></iframe>
     </div>

--- a/testing/it/http/src/test/resources/components/embed-v1-url-youtube.html
+++ b/testing/it/http/src/test/resources/components/embed-v1-url-youtube.html
@@ -1,7 +1,7 @@
 <div class="embed-v1 embed">
     <div data-cmp-data-layer="{&#34;embed_v1_url_youtube&#34;:{&#34;@type&#34;:&#34;core-component/components/embed-v1&#34;}}"
          id="embed_v1_url_youtube" class="cmp-embed">
-        <iframe width="200" height="113" src="https://www.youtube.com/embed/vpdcMZnYCko?feature=oembed" frameborder="0"
+        <iframe width="200" height="113" referrerpolicy="strict-origin-when-cross-origin" src="https://www.youtube.com/embed/vpdcMZnYCko?feature=oembed" frameborder="0"
                 allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen
                 title="Ocean Surface: 10 Hours of Relaxing Oceanscapes | BBC Earth"></iframe>
     </div>

--- a/testing/it/http/src/test/resources/components/embed-v2-url-trailing-whitespace.html
+++ b/testing/it/http/src/test/resources/components/embed-v2-url-trailing-whitespace.html
@@ -1,7 +1,7 @@
 <div class="embed-v2 embed">
     <div data-cmp-data-layer="{&#34;embed_v2_url_trailing_whitespace&#34;:{&#34;@type&#34;:&#34;core-component/components/embed-v2&#34;}}"
          id="embed_v2_url_trailing_whitespace" class="cmp-embed">
-        <iframe width="200" height="113" src="https://www.youtube.com/embed/vpdcMZnYCko?feature=oembed" frameborder="0"
+        <iframe width="200" height="113" referrerpolicy="strict-origin-when-cross-origin" src="https://www.youtube.com/embed/vpdcMZnYCko?feature=oembed" frameborder="0"
                 allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen
                 title="Ocean Surface: 10 Hours of Relaxing Oceanscapes | BBC Earth"></iframe>
     </div>

--- a/testing/it/http/src/test/resources/components/embed-v2-url-youtube.html
+++ b/testing/it/http/src/test/resources/components/embed-v2-url-youtube.html
@@ -1,7 +1,7 @@
 <div class="embed-v2 embed">
     <div data-cmp-data-layer="{&#34;embed_v2_url_youtube&#34;:{&#34;@type&#34;:&#34;core-component/components/embed-v2&#34;}}"
          id="embed_v2_url_youtube" class="cmp-embed">
-        <iframe width="200" height="113" src="https://www.youtube.com/embed/vpdcMZnYCko?feature=oembed" frameborder="0"
+        <iframe width="200" height="113" referrerpolicy="strict-origin-when-cross-origin" src="https://www.youtube.com/embed/vpdcMZnYCko?feature=oembed" frameborder="0"
                 allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen
                 title="Ocean Surface: 10 Hours of Relaxing Oceanscapes | BBC Earth"></iframe>
     </div>

